modules/microshift-ca-adding-bundle-ostree.adoc

```diff
@@ -0,0 +1,53 @@
+//Module included in the following assemblies:
+//
+//* microshift_install/microshift-embed-in-rpm-ostree.adoc
+
+
+:_content-type: PROCEDURE
+[id="microshift-ca-adding-bundle-ostree_{context}"]
+= Adding a certificate authority bundle to an ostree image
+You can include additional trusted certificate authorities (CAs) to the {op-system-ostree-first} `rpm-ostree` image by adding them to the blueprint you use to create the image. Using the following procedure sets up additional CAs to be trusted by the operating system when pulling images from an image registry.
+
+[NOTE]
+====
+The procedure requires you to configure the CA bundle customizations in the blueprint, then add steps to your kickstart file to enable the bundle. In the following steps, `data` is the key, `<value>` represents the PEM-encoded certificate.
+====
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+
+.Procedure
+
+. Add the following custom values to your blueprint to add a directory, `/etc/pki/ca-trust/source/anchors/`, and a file, `/etc/pki/ca-trust/source/anchors/cert1.pem`, to the image:
+
+.. Add instructions to your blueprint on the host where the image is built to create the directory for your certificate bundles:
++
+[source,terminal]
+----
+[[customizations.directories]]
+path = "/etc/pki/ca-trust/source/anchors"
+----
+
+.. Add instructions to your blueprint on the host where the image you are using has booted to create the certificate bundles:
```

Comment:
We probably only need to say this once.

Reply:
Okay I tweaked this to be one step. I think it might flow better that way.